site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md

```diff
@@ -0,0 +1,140 @@
+
+* [Raspberry PI OS Lite 64-bit](https://downloads.raspberrypi.org/raspios_lite_arm64/images/raspios_lite_arm64-2022-04-07/2022-04-04-raspios-bullseye-arm64-lite.img.xz)
+* [Raspberry PI OS Lite 32-bit](https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2022-04-07/2022-04-04-raspios-bullseye-armhf-lite.img.xz)
+
+Once you get the distribution, the installation procedure will be the same in both cases.
+
+Please note that in both cases you have to run the installation software with administrative permissions.
+
+There are various methods and different installation software that can be used for the installation but in this guide we'll make use of [Raspberry PI Imager](https://www.raspberrypi.com/software/)
+
+After installing the Raspberry PI imager according to your respective operating system. Launch the software with administrative permissions so you can be able to write into the SD card.
+
+![Raspberry pi image default screen](images/download.png)
+
+Then click the ***'CHOOSE OS'*** button , select ***'CUSTOM'*** from the drop down menu that will be displayed , then you can locate the Raspberry pi image you downloaded earlier.
+
+Now plug in your card reader with the micro SD card inserted into any of the various USB ports on your computer and click on the ***'CHOOSE STORAGE'*** button and locate your SD card in the options displayed (usually your sd card is the only option displayed if no other storage devices are plugged into your computer).
+
+The ***'WRITE'*** button at this point would change colour depicting that you're ready to write into your SD card , but before we write into the storage device click the 'SETTINGS' icon below the gear button to access options to pre-configure WiFi , SSH (recommended)  and so on and set according to your preferrences.
+
+Once this is done, we can now click the ***'WRITE'*** to write into our SD card .
+It doesn't take much time, once the raspberry pi imager is done with the writing process it displays the following prompt.
+
+![Raspberry pi imager writing complete](images/rpi_imager_write_completed.jpeg)
+
+Now all we have to do is unplug the SD card from the card reader and insert into the raspberry pi and connect it to a power source and wait for the booting process to finish up.
+
+## Step 2 : Connecting your raspberry pi to your android phone
+
+Next we turn on our router or mobile hotspot with the wifi setup preinstalled in Step1 to enable raspberry pi connect automatically to it and create a network in which both devices can share connectivity
+
+Using any ssh client , connect to the ip address(192.168.43.112 in my case) of raspberry pi on the network using the preconfigured ssh credentials in Step 1 to enable access to a terminal window
+   
+![Neofetch ssh screen](images/IMG_20220819_105041.jpg)
+
+Run the following commands to install a VNC server on the raspberry pi
+
+* Update the raspberry pi repository list
+
+```
```

Comment:
We usually use put "bash" after the 3 backquotes - that then uses white on black as per https://www.zaproxy.org/faq/how-can-you-import-post-requests/#proxy-another-tool - but its up to you. Either change none or all 😉